nodejs/eks/cluster.ts

```diff
@@ -627,6 +624,74 @@ export function createCore(
         },
     );
 
+    pulumi.output(args.fargate).apply((fargate) => {
```

Comment:
What do you think about lifting the addons out of the apply and always creating them?
If fargate is enabled we set the compute type config.

Reply:
Done